programmers/2일차/두-정수사이의-합.js

```diff
@@ -0,0 +1,13 @@
+function solution(a, b) {
+    const maxAB = Math.max(a,b);
+    const minAB = Math.min(a,b);
```

Comment:
```suggestion
    const maxNum = Math.max(a,b);
    const minNum = Math.min(a,b);
```
최댓값 최솟값을 의미하는 변수이기에  이게 더 직관적인 것 같아요 ㅎㅎ